modules/core/src/main/java/org/apache/ignite/internal/binary/BinaryEnumObjectImpl.java

```diff
@@ -473,4 +473,17 @@ public BinaryEnumObjectImpl(BinaryContext ctx, byte[] arr) {
 
         return reader.afterMessageRead(BinaryEnumObjectImpl.class);
     }
+
+    /** {@inheritDoc} */
+    @Override public int size() {
+        if (valBytes == null) {
+            try {
+                valBytes = U.marshal(ctx.marshaller(), this);
+            } catch (IgniteCheckedException e) {
```

Comment:
Please note that catch ( should start on a new line according to https://cwiki.apache.org/confluence/display/IGNITE/Coding+Guidelines

I will fix that during merge.

Reply:
Sorry about that, will ensure this henceforth :)